addons/fleet/models/fleet.py

```diff
@@ -143,7 +143,7 @@ class FleetVehicle(models.Model):
 
     def _get_default_state(self):
         state = self.env.ref('fleet.vehicle_state_active', raise_if_not_found=False)
-        return state and state.id or False
+        return state if state and state.id else False
```

Comment:
this is the same that `return state if state else False` and also the same that `return state or False`, both simpler 🤔